doc/source/whatsnew/v1.2.0.rst

```diff
@@ -801,6 +801,7 @@ Reshaping
 - Bug in :func:`merge_ordered` returned wrong join result when length of ``left_by`` or ``right_by`` equals to the rows of ``left`` or ``right`` (:issue:`38166`)
 - Bug in :func:`merge_ordered` didn't raise when elements in ``left_by`` or ``right_by`` not exist in ``left`` columns or ``right`` columns (:issue:`38167`)
 - Bug in :func:`DataFrame.drop_duplicates` not validating bool dtype for ``ignore_index`` keyword (:issue:`38274`)
+- Bug in :meth:`Series.isin` cast ``float`` unnecessarily to ``int`` when :class:`Series` to look in was from dtype ``int`` (:issue:`19356`)
```

Comment:
"cast ``float`` unnecessarily" -> "unnecessarily casting ``float`` dtypes"

Reply:
probably needs to move to 1.3.0

Reply:
I think it makes more sense removing the whatsnew entirely, since your fix went into 1.2. Would be confusing having this in 1.3